### PR TITLE
RET-3372 implementation

### PR DESCRIPTION
**Pull request by btensay**
Method for formatting claimant "Title" is refactored.  Unit tests updated as well.

Locally tested with unit test only. To be tested on perftest with Docmosis.  

https://tools.hmcts.net/jira/browse/RET-3372

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by hsjhita — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java [hunk 1]
```diff
@@ -70,12 +72,37 @@ public String claimantFullName() {
...
+        }
+
+        //Preferred title is set rather than the normal title
+        if (Strings.isNullOrEmpty(trimStringValue(claimantTitle))
+            && OTHER.equals(trimStringValue(claimantPreferredTitle))) {
+            return claimantTitleOther;
+        }
+
+        //If title is "Prefer Not to Say"
+        if (PREFER_NOT_TO_SAY.equals(trimStringValue(claimantTitle))) {
```
Comment (GauravChawlaMOJ) [inline src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java:95]: claimant preferred title will be prefer not to say and not claimantTitle

Comment (GauravChawlaMOJ) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java:95]: Please have all the checks for field claimantPreferredTitle too

Comment (btensay) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java:95]: About the first comment: From the screen shot on this ticket,  it is claimantTitle can be "Prefer not to say". Can both be "Prefer not to say"? Are they not exclusive? Bt that I mean they do not get set both at the same time.  

About the second comment: I check the conditions for claimantPreferredTitle too.

Comment (btensay) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/types/ClaimantIndType.java:95]: I add fix as discussed.

**Review by GauravChawlaMOJ — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**